config/samples/glance_v1beta1_glanceapi.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
 spec:
   serviceUser: glance
   serviceAccount: glance
-  containerImage: quay.io/podified-antelope-centos9/openstack-glance-api:current-podified
```

Comment:
Not sure functional tests are going to work with this [1] because the `webhook` is triggered by the `top-level` `CR`, which is not happening with this (/cc @abays to ack/nack what I'm saying)

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_glance-operator/358/pull-ci-openstack-k8s-operators-glance-operator-main-functional/1719366033648652288/artifacts/test/build-log.txt

Reply:
There is actually no webhook for GlanceAPI, only top level Glance. If we want to continue running tests on that sample we'd have to implement a webhook for it (which I think is beyond the scope of this patch). While some operators do implement webhooks on sub-resources it isn't strictly required I think as those resources should not be used externally anyway.

Reply:
I actually disabled the sample test for now. (if that works). If not, the approach should be to first do another PR to create webhooks on the GlanceAPI resource. Then once that is completed follow up with this PR again.

Reply:
I think it's ok to merge this patch as it is, and I can follow up by creating the missing webhook later. I see you already added a note for that, so I'm fine merging this as long as the tests are green. 
If I manage to push a webhooks related patch early next week, we can rebase this one on top of that.

Reply:
@dprince I think that in order to unblock this patch (in a more clean way) is to merge PR #363 first and rebase this one that will result in just removing the remaining lines.
Let me know what you think and eventually review #363.

Reply:
@fmount thanks! I approved the PR. Once it lands I can rebase. Also, I added a custom kuttl script at the end to verify the containerImage. It works now, but only verifies the top level "Glance" CR. It could be expanded to verify the sub-resources for GlanceAPI and also the Deployments that it creates.